projects/core/src/site-theme/store/site-theme-store.module.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { EffectsModule } from '@ngrx/effects';
+import { StoreModule } from '@ngrx/store';
+import {
+  StateConfig,
+  StateTransferType,
+} from '../../state/config/state-config';
+import { effects } from './effects/index';
+import { reducerProvider, reducerToken } from './reducers/index';
+import { SITE_THEME_FEATURE } from './state';
+import { provideDefaultConfigFactory } from '../../config/config-providers';
+
+export function siteThemeStoreConfigFactory(): StateConfig {
+  const config: StateConfig = {
+    state: {
+      ssrTransfer: {
+        keys: { [SITE_THEME_FEATURE]: StateTransferType.TRANSFER_STATE },
+      },
+    },
+  };
+  return config;
+}
```

Comment:
Did you test it with SSR enabled? 

I'm worrying that perhaps it might break taking the Theme from localStorage on page refresh. I guess the `default` theme calculated during the anonymous SSR rendering and transferred to CSR directly to the ngrx state will take precedence when page starts, and therefore a value from localStorage might be dismissed.

All that said, those just my worries from top of my head. But to be 100% sure, I'd like us to test this scenario it with SSR enabled and have a proof.

Reply:
tested SSR mode